scripts/configinverter/main.go

```diff
@@ -0,0 +1,132 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2025 Datadog, Inc.
+
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log/slog"
+	"os"
+	"sort"
+
+	"github.com/DataDog/dd-trace-go/v2/internal/env"
+	"github.com/dave/jennifer/jen"
+)
+
+func main() {
+	var (
+		input  string
+		output string
+	)
+	flag.StringVar(&input, "input", "./internal/env/supported-configurations.json", "Path to the input file")
+	flag.StringVar(&output, "output", "./internal/env", "Path to the output directory")
+	flag.Parse()
+
+	var err error
+	command := flag.Arg(0)
+	switch command {
+	case "generate":
+		err = generate(input, output)
+	case "check":
+		err = check(input)
+	default:
+		err = fmt.Errorf("unknown command: %s", command)
+	}
+
+	if err != nil {
+		slog.Error("error executing command", "error", err)
+		os.Exit(1)
+	}
+	slog.Info("success executing command", "command", command)
+}
+
+type supportedConfiguration struct {
+	SupportedConfigurations map[string][]string `json:"supportedConfigurations"`
+	Aliases                 map[string][]string `json:"aliases"`
+}
+
+func generate(input, output string) error {
+	keys, err := getSupportedConfigurationsKeys(input)
+	if err != nil {
+		return fmt.Errorf("error getting supported configuration keys: %w", err)
+	}
+
+	slog.Info("supported configuration keys count", "count", len(keys))
+
+	f := jen.NewFilePath(output)
+	f.HeaderComment("Unless explicitly stated otherwise all files in this repository are licensed")
+	f.HeaderComment("under the Apache License Version 2.0.")
+	f.HeaderComment("This product includes software developed at Datadog (https://www.datadoghq.com/).")
+	f.HeaderComment("Copyright 2025 Datadog, Inc.")
+	f.Comment("Code generated by github.com/DataDog/dd-trace-go/v2/scripts/configinverter. DO NOT EDIT.")
```

Comment:
```suggestion
	f.HeaderComment(`Unless explicitly stated otherwise all files in this repository are licensed\n
under the Apache License Version 2.0.\n"
This product includes software developed at Datadog (https://www.datadoghq.com/).\n
Copyright 2025 Datadog, Inc.\n
Code generated by github.com/DataDog/dd-trace-go/v2/scripts/configinverter. DO NOT EDIT.`)
```

I wonder if something like this would work?

Reply:
This is just a nitpick.

Reply:
Adding multiline comments creates a comment block that's why I went for multiple 1 liner calls

```
/*
Unless explicitly stated otherwise all files in this repository are licensed
under the Apache License Version 2.0.
This product includes software developed at Datadog (https://www.datadoghq.com/).
Copyright 2025 Datadog, Inc.

Code generated by github.com/DataDog/dd-trace-go/v2/scripts/configinverter. DO NOT EDIT.
*/

package env
```